### ISSUE TITLE
How do I get each Widget to be treated as an individual node in the Semantics tree? 

### ISSUE THREAD

**Issue by chris-moore-sonocent** [flutter/flutter#54725]
I have created a custom widget to represent a tile in a `ListView`, an example of which is included below. The tile contains an `Inkwell`, such that it can be tapped. The tile also contains a `MaterialButton`. I've wrapped everything in `Semantics` accordingly, but whenever Talkback is enabled and I tap the tile, the entire tile is highlighted. This makes the button within impossible to tap.

How do I get each `Widget` to be treated as an individual node in the Semantics tree? The `MergeSemantics` documentation (https://api.flutter.dev/flutter/widgets/MergeSemantics-class.html) seems to imply that `Widgets` should be treated individually unless they are wrapped in a `MergeSemantics`.

```
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Scaffold(
        body: ListView(
          children: [Tile()],
        ),
      ),
    );
  }
}

class Tile extends StatelessWidget {
  @override
  Widget build(BuildContext context) => Padding(
        padding: EdgeInsets.only(top: 12),
        child: InkWell(
          onTap: () => print('Tile tapped'),
          child: _tile(),
        ),
      );

  Widget _tile() => Container(
        padding: EdgeInsets.all(12),
        decoration: BoxDecoration(
          borderRadius: BorderRadius.all(Radius.circular(5)),
          border: Border.all(color: Colors.grey, width: 1),
          shape: BoxShape.rectangle,
        ),
        child: Row(
          children: [
            _textAndCount(),
            SizedBox(width: 12),
            _button(),
          ],
        ),
      );

  Widget _textAndCount() => Expanded(
        child: Column(
          children: [
            SizedBox(
              width: double.infinity,
              child: _text(),
            ),
            _count(),
          ],
        ),
      );

  Widget _text() => Text(
        'text',
        overflow: TextOverflow.ellipsis,
        maxLines: 2,
        textAlign: TextAlign.start,
      );

  Widget _count() =>
      Padding(
        padding: EdgeInsets.only(top: 12),
        child: Row(
          children: [
            Semantics(
              label: '0 things',
              child: ExcludeSemantics(
                child: Container(
                  padding: EdgeInsets.symmetric(horizontal: 12),
                  decoration: BoxDecoration(
                    shape: BoxShape.rectangle,
                    color: Colors.grey,
                    borderRadius: BorderRadius.circular(10),
                  ),
                  child: Text('0'),
                ),
              ),
            ),
          ],
        ),
      );

  Widget _button() => Semantics(
        label: 'button',
        button: true,
        child: Tooltip(
          message: 'button',
          excludeFromSemantics: true,
          child: ExcludeSemantics(
            child: MaterialButton(
              onPressed: () => print('Button tapped'),
              child: Container(
                child: SizedBox(
                  height: 44,
                  width: 44,
                  child: Icon(Icons.add),
                ),
                decoration: BoxDecoration(
                  border: Border.all(color: Colors.grey, width: 2),
                  shape: BoxShape.circle,
                ),
              ),
            ),
          ),
        ),
      );
}
```

**Comment by TahaTesser** [comment]
Tested on Samsung, only ListTile is selected

<details>
<summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel dev, v1.18.0-dev.5.0, on Mac OS X 10.15.4 19E287, locale
    en-GB)
    • Flutter version 1.18.0-dev.5.0 at /Users/taha/Code/flutter_dev
    • Framework revision 7f56b53de4 (2 days ago), 2020-04-12 12:00:01 -0400
    • Engine revision beb8a7ec48
    • Dart version 2.8.0 (build 2.8.0-dev.20.0 89b0f67261)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/taha/Code/sdk
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/taha/Code/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.
[✓] Xcode - develop for iOS and macOS (Xcode 11.4)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4, Build version 11E146
    • CocoaPods version 1.9.1
[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build
      1.8.0_212-release-1586-b4-5784211)
[✓] VS Code (version 1.44.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.9.1
[✓] Connected device (4 available)
    • SM M305F   • 32003c30dc19668f • android-arm64  • Android 10 (API 29)
    • macOS      • macOS            • darwin-x64     • Mac OS X 10.15.4 19E287
    • Chrome     • chrome           • web-javascript • Google Chrome
      81.0.4044.92
    • Web Server • web-server       • web-javascript • Flutter Tools
• No issues found!
```

</details>

**Comment by goderbauer** [comment]
If you want a thing to be a separate semantics node you can always wrap it in `Semantics(container: true, child: ...)`.

This question is probably better suited for stack overflow. Closing.
